e2e/network_test.go

```diff
@@ -263,7 +264,7 @@ var wgetPodNT = &v1.Pod{
 			{
 				Name:    "wget-container",
 				Image:   "busybox:1.26",
-				Command: []string{"wget", "--timeout", "5", "nginx-service-nt"},
+				Command: []string{"wget", "--timeout", "5", "--retry-connrefused", "--waitretry", "1", "--tries", "5", "nginx-service-nt"},
```

Comment:
Unfortunate. We need to re-think this logic somehow -- retrying on checking the pod status doesn't make a lot of sense (if it fails once it's not going to change the status).